api/api.proto

```diff
@@ -155,4 +180,6 @@ service Dex {
   //
   // Note that each user-client pair can have only one refresh token at a time.
   rpc RevokeRefresh(RevokeRefreshReq) returns (RevokeRefreshResp) {};
+
```

Comment:
Add comment.

BTW, why do you need this, isn't refresh token already returned?